common/scala/src/main/scala/whisk/core/containerpool/ContainerFactory.scala

```diff
@@ -44,6 +44,26 @@ trait ContainerFactory {
 
   /** cleanup any remaining Containers; should block until complete; should ONLY be run at startup/shutdown */
   def cleanup(): Unit
+
```

Comment:
This comes directly from the "old" ContainerPool:
https://github.com/apache/incubator-openwhisk/blob/e47cc71bcd73ebb3da352eeadcc4236630680e92/core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala#L826-L844

which was deleted in
https://github.com/apache/incubator-openwhisk/commit/10fae73e15cf886bb3ad892e659b15a28cc4c80b